en/admin/config.rst

```diff
@@ -2303,6 +2311,20 @@ The following are other parameters. The type and value range for each parameter
 
 **vacuum_ovfp_check_duration** specifies the duration for which data related to the count of index overflow pages, gathered by vacuum threads, is retained. Data that remains unchanged within the specified duration will be automatically removed. The unit of it's value is minutes.
 
+**deduplicate_key_level**
+
+ Specifies the *deduplicate level* value implicitly specified in the index creation statement. Default is -1. See :doc:`/sql/schema/table_stmt`\ and :doc:`/sql/schema/index_stmt`.
```

Comment:
```suggestion
 specifies the *deduplicate level* value implicitly specified in the index creation statement. Default is -1. See :doc:`/sql/schema/table_stmt`\ and :doc:`/sql/schema/index_stmt`.
```

Reply:
@mhoh3963 
Shouldn't the first letter of a sentence be capitalized?

Reply:
What about "It specifies ..."